README.md

```diff
@@ -817,27 +817,20 @@ customButtons: [{
 
 - **customButtons.click** - Function that is called when the button is clicked
 
-## Adding Registration Link
+## Adding Registration Button
 
-You can add registration link underneath the login form on the primary auth page by setting the following config options. If you'd like to change the label text and link text use the `i18n` config option.
+You can add a registration button to the primary auth page by setting the following config options. If you'd like to change the text, use the i18n config option
```

Comment:
Is there a reason you removed the period at the end of the sentence, and removed the `i18n` backticks? It's now inconsistent with the other sentences in the readme.